cmd/podman/common/specgen.go

```diff
@@ -527,6 +527,8 @@ func FillOutSpecGen(s *specgen.SpecGenerator, c *ContainerCLIOpts, args []string
 			}
 
 			switch con[0] {
+			case "procfs-opts":
```

Comment:
nit: Lets call this "procopts" or "proc-opts"
People understand proc no need for fs.